docs/cimpl6LanguageReference.md

```diff
@@ -1057,124 +1044,127 @@ With FHIR as the target output, the right-hand side of the map statement can poi
-| dotted path | **simple element** |_Value.ActiveFlag maps to **active**_ |
-| dotted path | **dotted path** | _AdverseReaction.AllergenIrritant maps to **reaction.substance**_|
-| `Value Choice` path | **simple element** | _BodySiteOrCode.concept maps to **bodySite**_ |
+| `Property` reference without path | **simple element** | `MaritalStatus maps to maritalStatus` |
+| `Property` reference without path | **extension** | `Ethnicity maps to http://hl7.org/fhir/us/core/StructureDefinition/us-core-ethnicity` |
+| dotted path | **simple element** |`Value.ActiveFlag maps to active` |
+| dotted path | **dotted path** | `AdverseReaction.AllergenIrritant maps to reaction.substance`|
+| `Value Choice` path | **simple element** | `BodySiteOrCode.concept maps to bodySite` |
 
->**Note:** The mapping grammar version is 5.1; the bracketed value syntax cannot be used in paths.  This is why the above example uses _BodySiteOrCode.concept_ rather than _BodySiteOrCode[concept]_.
+>**Note:** The mapping grammar version is 5.1; the bracketed value syntax cannot be used in paths.  This is why the above example uses `BodySiteOrCode.concept` rather than `BodySiteOrCode[concept]`.
 
 ### Mapping to References
 
-FHIR **resources** and **profiles** require an explicit **Reference()** indicator when referring to another **Resource** or **Profile**. In CIMPL, whenever a `Property` is an `Entry`, that `Property` is mapped as a **reference** in FHIR. There is no need to create an explicit **Reference()**.
-
-Example:
-
-| CIMPL Definition | Corresponding FHIR |
-|----------|---------|
-| `Property`:  _Encounter 0..1_ | _**encounter:    0..1  Reference(Encounter)**_ |
+FHIR **resources** and **profiles** require an explicit [**Reference()** indicator](https://www.hl7.org/fhir/references.html) when referring to another **Resource** or **Profile**. In CIMPL, whenever a `Property` is an `Entry`, that `Property` is mapped as a **Reference** in FHIR. There is no need to create an explicit **Reference()**.
 
 ### Placement of Extensions
 
 As mentioned above, if no mapping rule is found for a `Property`, an **extension** to the FHIR **resource** or **profile** will be automatically created. By default, the **extension** will be a top-level **element** in the FHIR **profile**. Sometimes you may want the **extension** to appear in a different place, perhaps inside a nested **element**. To put the **extension** precisely where you want it, you can map to FHIR paths that include `extension`.
 
 Here's an example involving the FHIR resource [AdverseEvent](https://www.hl7.org/fhir/adverseevent.html):
 
-_PotentialCause.CauseCategory `maps to` **suspectEntity.extension**_
+`PotentialCause.CauseCategory maps to suspectEntity.extension`
 
 This mapping rule will create an **extension** named _**CauseCategory**_ for **AdverseEvent.suspectEntity**. Rules such as:
 
-_DetectionTime `maps to` **extension**_
+`DetectionTime maps to extension`
```

Comment:
extension should be bolded, this is a reference to a FHIR extension.

Reply:
No, same as previous comment.